js/renovation/pager/resizable-container.tsx

```diff
@@ -0,0 +1,169 @@
+import {
+  Component, ComponentBindings, JSXComponent,
+  Ref, Effect, Template, InternalState, Event,
+} from 'devextreme-generator/component_declaration/common';
+import getElementComputedStyle from './get-computed-style';
+import resizeCallbacks from '../../core/utils/resize_callbacks';
+import PagerProps from './pager-props';
+// bug in generator: Max call stack
+// import { TwoWayProps } from './pager-content';
+// import type PagerContentProps from './pager-content';
```

Comment:
Is this comments actual?

Reply:
Maybe. I planed to review all generator issues then they be fixed